packages/server/src/index.ts

```diff
@@ -1460,6 +1534,28 @@ export class App {
         let toDate
         if (endDate) toDate = setDateToStartOrEndOfDay(endDate, 'end')
 
+        if (feedback) {
+            const messages = await this.AppDataSource.getRepository(ChatMessage)
+                .createQueryBuilder('chat_message')
+                .leftJoinAndMapOne('chat_message.feedback', ChatMessageFeedback, 'feedback', 'feedback.messageId = chat_message.id')
+                .where('chat_message.chatflowid = :chatflowid', { chatflowid })
+                .andWhere(chatType ? 'chat_message.chatType = :chatType' : 'TRUE', { chatType })
+                .andWhere(chatId ? 'chat_message.chatId = :chatId' : 'TRUE', { chatId })
+                .andWhere(memoryType ? 'chat_message.memoryType = :memoryType' : 'TRUE', {
+                    memoryType: memoryType ?? (chatId ? IsNull() : undefined)
+                })
+                .andWhere(sessionId ? 'chat_message.sessionId = :sessionId' : 'TRUE', {
+                    sessionId: sessionId ?? (chatId ? IsNull() : undefined)
+                })
+                .andWhere(fromDate && toDate ? 'chat_message.createdDate = :createdDate' : 'TRUE', {
```

Comment:
filtering by date is not working anymore, please check.
<img width="881" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/526eda58-bab4-4f7e-9a26-55e109162078">

Reply:
Fixed this issue.